core/ecschema-metadata/src/Context.ts

```diff
@@ -3,16 +3,29 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
+import { DelayedPromise } from "./DelayedPromise";
 import { SchemaMatchType } from "./ECObjects";
 import { ECObjectsError, ECObjectsStatus } from "./Exception";
 import { MutableSchema, Schema } from "./Metadata/Schema";
 import { SchemaItem } from "./Metadata/SchemaItem";
 import { SchemaItemKey, SchemaKey } from "./SchemaKey";
 
 /**
- * @beta
+ * Holds partially-loaded schema and corresponding promise to fully load the schema
+ * @alpha
+ */
+interface LoadingSchema {
+  schema: Schema;
+  loadSchemaPromise: DelayedPromise<Schema>;
+}
+/**
+ * @alpha
+ */
+export class LoadedSchemas extends Array<Schema> { }
+/**
+ * @alpha
  */
-export class SchemaMap extends Array<Schema> { }
+export class LoadingSchemas extends Array<LoadingSchema> { }

```

Comment:
You're exporting a type that is an array of a non-exported type. Did you intend to export `LoadingSchema`?

Reply:
It looks to me like the only usage of `LoadedSchemas` and `LoadingSchemas` is as `private` fields, in which case I don't see a reason to export either of them. Did I overlook something?

Reply:
Your observation is correct.  Currently, I intend to use `LoadedSchemas` and `LoadingSchemas` as private members of the `SchemaCache` class. As such, I'll remove the export modifier from them.

Reply:
Removed the export modifiers from `LoadedSchemas` and `LoadingSchemas`